src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,31 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
```

Comment:
Your `grid-column-width` is bigger than `card-width`, I think in this task we have to use the same width (card-width = column-width), and it have to work with different widths (`@media`). But in task I didn't see it requirement